### PR TITLE
Updates descriptions of enrollment tracks

### PR DESCRIPTION
**Pull request by lamagnifica**
## [DOC-3038](https://openedx.atlassian.net/browse/DOC-3038)

Insights now reports enrollment tracks other than Honor, Verified, and Professional. Updates to an image and to the descriptions of how enrollment is reported are included. 
### Date Needed

13 July 2016
### Reviewers

Possible roles follow. PR submitter checks the boxes after each reviewer finishes and gives :+1:. 
- [x] Subject matter expert: @dsjen 
- [x] Subject matter expert: @thallada 
- [x] Doc team review (sanity check): @pdesjardins or @srpearce 
### Testing
- [x] Ran make html without warnings or errors
### Post-review
- [x] Add description to release notes task as a comment
- [x] Squash commits

### CODE REVIEW

File: docs/en_us/dashboard/source/Reference.rst [hunk 1]
```diff
@@ -89,32 +89,38 @@ For information about viewing enrollment activity data in edX Insights, see
...
 
-  In addition to honor code certificates, courses can offer a verified
-  certificate option or enroll students through a professional education
-  program.
-
 * Each enrolled student is included in one, and only one, of the possible
   enrollment tracks on a given date. The student's enrollment track as of 23:59
   UTC is used each day.
 
 **Enrollment Over Time report**
```
Comment (pdesjardins) [inline docs/en_us/dashboard/source/Reference.rst:96]: Capitalize "report" in the title.

File: docs/en_us/dashboard/source/Reference.rst [hunk 2]
```diff
@@ -89,32 +89,38 @@ For information about viewing enrollment activity data in edX Insights, see
...
-  report includes columns for **Verified Enrollment** and **Honor Code
-  Enrollment**.
+This report includes a column for each enrollment option or certification track
+offered by the course. The columns that can appear for edx.org courses follow.
+
+* The **Audit** column reports the count of learners enrolled in the audit
+  track.
+
+  * For courses offered after December 2015, the audit enrollment option is not
+    certificate eligible.
```
Comment (pdesjardins) [inline docs/en_us/dashboard/source/Reference.rst]: "Audit enrollment option is not certificate eligible" is confusing for me. Learners who choose the audit enrollment option are not eligible for certificates?

File: docs/en_us/dashboard/source/Reference.rst [hunk 3]
```diff
@@ -89,32 +89,38 @@ For information about viewing enrollment activity data in edX Insights, see
...
+
+  * For courses that ran between August 2013 and October 2014, a certificate-
+    eligible audit track option was available. Learners who enrolled in the
+    audit track and passed the course could request an honor certificate.
```
Comment (pdesjardins) [inline docs/en_us/dashboard/source/Reference.rst]: Number agreement: could request an honor certificate > could request honor certificates

File: docs/en_us/dashboard/source/Reference.rst [hunk 4]
```diff
@@ -89,32 +89,38 @@ For information about viewing enrollment activity data in edX Insights, see
...
+
+* The **Verified** column reports the count of learners enrolled in the
+  verified track. This count does not include learners who have made the
+  additional election to receive credit (if the course offers credit).
```
Comment (pdesjardins) [inline docs/en_us/dashboard/source/Reference.rst]: election > choice, for simpler English?

File: docs/en_us/dashboard/source/Reference.rst [hunk 5]
```diff
@@ -89,32 +89,38 @@ For information about viewing enrollment activity data in edX Insights, see
...
 
-* If a course enrolls students through a professional education program, the
-  report includes columns for **Professional Enrollment** and **Honor Code
-  Enrollment**.
+* The **Verified with Credit** column reports the count of learners enrolled in
+  the verified track who have exercised the option to receive credit if they
```
Comment (pdesjardins) [inline docs/en_us/dashboard/source/Reference.rst]: exercised the option to > chosen to, to be simpler?

File: docs/en_us/dashboard/source/enrollment/Enrollment_Activity.rst [hunk 6]
```diff
@@ -32,15 +32,9 @@ represents the number of learners enrolled in the course on a particular date.
...
-
-* Honor certificate
-* Verified certificate
-* Professional education program
-
-Moving your cursor over the chart shows a tooltip with the counts for each
-enrollment type, and the current enrollment, for each day.
+The chart includes each of the enrollment options and tracks that are offered
+for your course. Moving your cursor over the chart shows a tooltip with the
+counts for each enrollment type, and the current enrollment, for each day.
```
Comment (pdesjardins) [inline docs/en_us/dashboard/source/enrollment/Enrollment_Activity.rst:37]: Just curious: is there a non-mouse option for accessibility, that can be mentioned here?

File: docs/en_us/dashboard/source/enrollment/Enrollment_Activity.rst [hunk 7]
```diff
@@ -62,18 +56,17 @@ The chart does reveal several periods when the rate of new enrollments
...
 automated enrollment events, or might want to research possible explanations.
-After the course start date, on September 17, 2014, enrollment continued to
+After the course start date, on 17 September 2014, enrollment continued to
 increase.
 
 The second example shows the Daily Student Enrollment chart for a small,
 private online course. In this course, the course team used the Instructor
-Dashboard to automatically enroll all of the students.
+Dashboard to automatically enroll all of the students over a couple of days'
+time.
```
Comment (pdesjardins) [inline docs/en_us/dashboard/source/enrollment/Enrollment_Activity.rst]: all of the students over a couple of days' time > over two days? Over several days?

**Comment by pdesjardins** [comment]
I added some thoughts, 👍 !

File: docs/en_us/dashboard/source/enrollment/Enrollment_Activity.rst (hunk 6 above)
Comment (dsjen) [inline docs/en_us/dashboard/source/enrollment/Enrollment_Activity.rst:37]: Unfortunately there is no non-mouse option.  We went through an initial a11y review and the recommendation was to hide the chart for screens and ensure tables are included, which is available in the enrollment page.

**Comment by dsjen** [comment]
👍

**Comment by thallada** [comment]
:+1:

File: docs/en_us/dashboard/source/Reference.rst (hunk 1 above)
Comment (lamagnifica) [inline docs/en_us/dashboard/source/Reference.rst:96]: I'm going to make a follow on PR to take care of this throughout the Insights guide (Insights slightly predated our standard to title cap all headings). I also want to convert these bolded "headers" to actual rst heading levels.

**Comment by lamagnifica** [comment]
thanks for the untangling support, @pdesjardins. New commit ready.

**Comment by thallada** [comment]
Is this ready to be merged?

**Comment by lamagnifica** [comment]
@thallada it is ready to be merged! however, I typically do not merge documentation updates until we are pretty close to releasing the software. This helps us avoid situations where all of the guides in readthedocs get rebuilt to include merged changes, even if the software hasn't been released yet...

File: docs/en_us/dashboard/source/enrollment/Enrollment_Activity.rst (hunk 6 above)
Comment (lamagnifica) [inline docs/en_us/dashboard/source/enrollment/Enrollment_Activity.rst:37]: to be a little more precise, for vision-impaired people, the table or download provides the data. For people who can't use a mouse, other means of changing focus are supported.

**Comment by lamagnifica** [comment]
Updated with new image files to show the changed colors for the enrollment activity graph. Only the latest commit needs to be reviewed, though other comments are welcome!
- [x] Subject matter expert: @dsjen 
- [x] Subject matter expert: @ajpal 
- [x] Doc team review (sanity check): @pdesjardins @catong  @srpearce

File: docs/en_us/dashboard/source/enrollment/Enrollment_Activity.rst [hunk 8]
```diff
@@ -56,24 +50,21 @@ course was created in Studio.
  :alt: A chart with the periods when the rate of enrollment was greater
```
Comment (catong) [inline docs/en_us/dashboard/source/enrollment/Enrollment_Activity.rst]: Wording of the alt text is slightly odd: "was greater" (than what?). Suggest "when the rate of enrollment increased..."

File: docs/en_us/dashboard/source/enrollment/Enrollment_Activity.rst [hunk 9]
```diff
@@ -56,24 +50,21 @@ course was created in Studio.
...
        circled.
 
-.. ColumbiaX/HIST1.1x/3T2014/enrollment/activity/
-
 The chart does reveal several periods when the rate of new enrollments
 increased rapidly, or "spiked" (circled). The team for this course might have
 the contextual knowledge to correlate those periods with marketing efforts or
 automated enrollment events, or might want to research possible explanations.
-After the course start date, on September 17, 2014, enrollment continued to
+After the course start date, on 17 September 2014, enrollment continued to
```
Comment (catong) [inline docs/en_us/dashboard/source/enrollment/Enrollment_Activity.rst]: Do you mention the year of the course example elsewhere? If I'm looking at just the chart and reading the example scenario description, year isn't required. If that's the case consider removing it from the description here - its inclusion just made me look hard at the chart to see if I'd missed it; made me doubt for a second whether I was looking at the correct chart.

Comment (catong) [inline docs/en_us/dashboard/source/enrollment/Enrollment_Activity.rst]: Also consider adding an arrow/caption at the 9/17 point in the chart to make it clear which of the 3 circled areas you are referring to?

**Comment by dsjen** [comment]
👍

**Comment by catong** [comment]
@lamagnifica Sanity checked chart against text. A couple of questions/suggestions that are optional. 👍

File: docs/en_us/dashboard/source/enrollment/Enrollment_Activity.rst [hunk 10]
```diff
@@ -47,15 +47,15 @@ certainly the course creator and another course team member) on the day the
...
        circled.
 
 The chart does reveal several periods when the rate of new enrollments
 increased rapidly, or "spiked" (circled). The team for this course might have
 the contextual knowledge to correlate those periods with marketing efforts or
 automated enrollment events, or might want to research possible explanations.
-After the course start date, on 17 September 2014, enrollment continued to
-increase.
+After the final spike, which coincided with the course start date on 17
+September, enrollment continued toincrease.
```
Comment (catong) [inline docs/en_us/dashboard/source/enrollment/Enrollment_Activity.rst]: @lamagnifica missing space between "to" and "increase"

**Comment by ajpal** [comment]
👍 👍

**Pull request merged**